source/features/batch-mark-files-as-viewed.tsx

```diff
@@ -66,7 +66,7 @@ function init(): void {
 	// `mousedown` required to avoid mouse selection on shift-click
 	delegate(document, '.js-toggle-user-reviewed-file-form', 'mousedown', batchToggle);
 	delegate(document, '.js-toggle-user-reviewed-file-form', 'submit', remember);
-	delegate(document, '.js-reviewed-checkbox', 'click', onAltClick);
+	delegate(document, '.js-toggle-user-reviewed-file-form', 'click', onAltClick);
```

Comment:
@yakov Can you change this (and probably the other selectors, too) to `.js-reviewed-toggle` please, as described in https://github.com/sindresorhus/refined-github/issues/4155#issuecomment-808500720?

Reply:
```suggestion
	delegate(document, '.js-reviewed-toggle', 'click', onAltClick);
```